src/utils/helpers.utils.js

```diff
@@ -181,3 +174,9 @@ export const isValidUuid = async identifier => {
   }
   return true;
 };
+export const serverError = (res, statusCode = 500) =>
```

Comment:
Hey @eben-k, am sure this function was created by @igbominadeveloper in his PR, can we use that implementation instead.